mmseg/models/backbones/mobile_net_v2.py

```diff
@@ -0,0 +1,57 @@
+from mmcv.cnn import ConvModule, build_norm_layer
+from torch import nn
+
+
+class InvertedResidual(nn.Module):
+
+    def __init__(self,
+                 inp,
```

Comment:
We may consider writing as mmcls [here](https://github.com/open-mmlab/mmclassification/blob/master/mmcls/models/backbones/mobilenet_v2.py#L14).